website/src/_data/capability-matrix.yml

```diff
@@ -509,10 +543,14 @@ categories:
             l1: 'Partially'
             l2: non-merging windows
             l3: State is supported for non-merging windows. SetState and MapState are not yet supported.
-          - class: spark
+          - class: spark-rdd
+            l1: 'Yes'
```

Comment:
It should be `Partially` since it's supported only in batch mode.

Reply:
+1